### PR TITLE
Python3 compat

### PR DESCRIPTION
**Pull request by bargst**
Small changes required to make pystun usable with python3

### CODE REVIEW

File: stun/cli.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function
```
Comment (Renelvon) [inline stun/cli.py:1]: Why is this line deleted? It causes no problems with Python 3.x.

Comment (bargst) [inline stun/cli.py:1]: It is a 2to3 modification, should I delete it ?
